streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStoreTest.java

```diff
@@ -60,23 +70,31 @@
     private final String storeName = "bytes-store";
     private RocksDBSegmentedBytesStore bytesStore;
     private File stateDir;
-    private final SessionKeySchema schema = new SessionKeySchema();
+    private final long windowSizeForTimeWindow = 500L;
```

Comment:
See my other comment: we can just declare three windows object as part of the test, and initialize them accordingly in `before`.